notebooks/tutorial_RGDR.ipynb

```diff
@@ -8,21 +8,52 @@
     "This notebook outlines the current status of the RGDR implementation."
```

Comment:
<p>Perhaps it would be better to keep this as <code>lag=1</code> As the lag is the (interval) difference between the 1st target and the precursors.</p><p> </p><p>Alternatively we can make users pass <code>i_interval</code> instead to avoid this ambiguity, but then there is still the issue of "what is lag" when having <em>two</em> target periods. See <a href="https://github.com/AI4S2S/s2spy/issues/56" target="_blank">#56</a>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/s2spy/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Thanks for the comment, Bart! Indeed I was a bit hesitant when tweaking this, as the selection of minus value sounds a bit weird. And I think it makes sense to call it "lag 1" or "i_interval -1". The "i_interval" reference system helps a bit for the issue of "what is lag" when multiple target periods are included. </p><p> </p><p>Anyway, let's discuss this.</p><p> </p><p> </p>

Reply:
Following the discussion, to be consistent and to avoid ambiguity, we only use `i_interval` to refer to the index.